enterprise/server/backends/userdb/userdb.go

```diff
@@ -783,7 +783,7 @@ func (d *UserDB) GetGroupUsers(ctx context.Context, groupID string, statuses []g
 
 	q := query_builder.NewQuery(`
 			SELECT u.user_id, u.email, u.first_name, u.last_name, ug.membership_status, ug.role
-			FROM "Users" AS u JOIN "UserGroups" AS ug`)
+			FROM "Users" AS u NATURAL JOIN "UserGroups" AS ug`)
```

Comment:
Does postgres not like the query w/o this?

Reply:
Assuming the answer is yes, I would suggest adding an explicit `ON` clause rather than relying on implicit column matching.

Reply:
It does not; I can use an `ON` clause.